### PR TITLE
Apply Jupyterhub DB changes to Test KfDef

### PR DESCRIPTION
**Pull request by gmfrasca**

### CODE REVIEW

File: tests/setup/kfctl_openshift.yaml [hunk 1]
```diff
@@ -14,10 +14,20 @@ spec:
       parameters:
         - name: s3_endpoint_url
           value: "s3.odh.com"
+        - name: notebook_destination
+          value: rhods-notebooks
```
Comment (LaVLaS) [inline tests/setup/kfctl_openshift.yaml:18]: @crobby This parameter is set in [odh-deployer(https://github.com/red-hat-data-services/odh-deployer/blob/main/opendatahub.yaml#L16) but it breaks when you run the jupyterhub CI tests manually.  Does openshift-ci use this kfdef for the CI runs or is there a RHODS specific tests kfdef?

Comment (crobby) [reply tests/setup/kfctl_openshift.yaml:18]: I need to double check, but I think it's using this kfdef

Comment (LaVLaS) [reply tests/setup/kfctl_openshift.yaml:18]: Local test runs work without it so I think it will probably be required for RHODS CI runs in openshift-ci

**Review by LaVLaS — APPROVED** [review]
/lgtm

Deploying and running the CI locally worked for me

**Review by crobby — APPROVED** [review]

**Pull request merged**